### DISCUSSION TITLE
Achievement Spotlight

### DISCUSSION THREAD

**Discussion by mosquitobird11** [RetroAchievements/RAWeb#1207] [Ideas]
It would be nice if we could "pin" a collection of achievements that we want to spotlight on our profiles, like so:

![image](https://user-images.githubusercontent.com/10202766/197909038-7bf41f49-6565-402c-aa0f-6bbe61e62fa3.png)
![image](https://user-images.githubusercontent.com/10202766/197909140-6bd5ed2d-391f-4e48-88a9-f5670abede79.png)

And you could pin / unpin these from the achievement lists they are from:
![image](https://user-images.githubusercontent.com/10202766/197909193-af69c1f2-2e2e-4069-89aa-566040f44880.png)

I tried to make this concept reusing as much css and html as I could, it just requires that the profile be split into a flexbox with a right and left side, and the profile image be added to the right side.

**Comment by Xeinok-Pals** [comment]
Along these lines, it might be a cool gamification to award a selection of profile banners or background images for some extreme challenges (maybe Quest system rewards in the future?)

**Comment by LuneDeCuivre** [comment]
I think this is a great idea! Perhaps quests could unlock displaying an achievement of a specific category with a custom funky border as well.
